Gemfile

```diff
@@ -22,7 +22,7 @@ gem "state_machines-mongoid"
 gem "uglifier"
 
 group :development, :test do
-  gem "database_cleaner"
+  gem "database_cleaner", "1.8.5"
```

Comment:
If the fix doesn't look simple - should we add a card to update it at a later point?

Reply:
Yes, we should definitely card it. I'll add to our backlog.

Reply:
https://trello.com/c/0DmAxikJ/1285-update-database-cleaner-in-imminence

Reply:
Awesome - thanks!